tree/10_ncf_internals/initialization.cf

```diff
@@ -0,0 +1,44 @@
+#####################################################################################
+# Copyright 2016 Normation SAS
+#####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
+
+# Initialization
+# This bundle will make necessary step to prepare ncf before running any configuration
+# It needs to be called just after loading ncf and its configuration.
+bundle agent init
```

Comment:
Should we reuse the `init` bundle? Given its name, it seems it may have been placed there for this kind of thing, but we may want to just create a new bundle for this.